lib/matplotlib/_fontconfig_pattern.py

```
@@ -98,11 +98,7 @@ def parse_fontconfig_pattern(pattern):
     for prop in parse.get("properties", []):
         if len(prop) == 1:
             if prop[0] not in _CONSTANTS:
-                _api.warn_deprecated(
-                    "3.7", message=f"Support for unknown constants "
-                    f"({prop[0]!r}) is deprecated since %(since)s and "
-                    f"will be removed %(removal)s.")
-                continue
```

This should raise an error now.